packages/block-library/src/search/edit.js

```diff
@@ -14,7 +14,7 @@ export default function SearchEdit( { className, attributes, setAttributes } ) {
 				aria-label={ __( 'Label text' ) }
 				placeholder={ __( 'Add label…' ) }
 				keepPlaceholderOnFocus
-				formattingControls={ [] }
+				disallowFormats
```

Comment:
Why prevent the user from having formatting in the label? `<a>`, `<b>`, `<i>`, etc. are allowed inside a `<label>`.

Reply:
Changed now.